src/components/NavBar.jsx

```diff
@@ -0,0 +1,45 @@
+import { Link } from "react-router-dom";
+import { Flex, Text, Icon } from "@chakra-ui/react";
+import { PlusSquareIcon } from "@chakra-ui/icons";
+import { FaHome, FaUserAlt } from "react-icons/fa";
+
+const NavBar = () => {
+  return (
+    <Flex as="nav" style={styles.navbar}>
```

Comment:
I think U can pass props to the parent flex for the nav with row direction, and then add the 3 buttons inside the flex, with space between, align-content to center.

currently you are adding 1 flex to 1 one button which is quite repetitive, but it works!